### ISSUE TITLE
Thank you for creating this!

### ISSUE THREAD

**Issue by tstout** [xwmx/nb#107]
I have been using this for a few month now. It is so useful!

**Comment by xwmx** [comment]
@tstout Glad you like it, and thanks for letting me know! 😄

**Comment by slothbear** [comment]
Agreed! I made it a resolution to embrace the nb lifestyle this year. It is going very well.

**Comment by xwmx** [comment]
Good to hear! I’m really enjoying `nb`, too. 😁

**Comment by dncnmcdougall** [comment]
This is wonderful. I have written my own Zettlekasten like store for embedding in nvim. But this looks like it will do everything I want, and has some additional features.
Really good thank you.
